guides/common/modules/con_upgrading-prerequisites.adoc

```diff
@@ -16,6 +16,11 @@ Upgrading {SmartProxy} takes approximately 10{range}30 minutes.
 
 * Ensure that you have sufficient storage space on your server.
 For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
+ifndef::foreman-deb[]
+* Ensure that you have at least the same amount of free space as that consumed by `/var/lib/pgsql/data`.
```

Comment:
The attribute is useful here too, actually:
```suggestion
* Ensure that you have at least the same amount of free space as that consumed by `{postgresql-lib-dir}/data`.
```

Reply:
There's `postgresql-data-dir` for `{postgresql-lib-dir}/data` already. If you want to be explicit, you could even write
```suggestion
* Ensure that you have at least the same amount of free space on `{postgresql-lib-dir}` as that consumed by `{postgresql-data-dir}`.
```

Otherwise people could check for free space on `/` while they have a `/var/lib/pgsql` partition.